src/components/link/link.hbs

```diff
@@ -6,9 +6,8 @@
 -->
 
 <a href="#" class="{{@root.prefix}}--link">Link</a>
-<a class="{{@root.prefix}}--link">Placeholder link</a>
-<p class="{{@root.prefix}}--link--disabled">Disabled Link</p>
-<br>
 <a href="#" class="{{@root.prefix}}--link">Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti explicabo
   nobis consequatur quod qui quam laboriosam placeat repudiandae. Ducimus ea aut omnis asperiores mollitia
   necessitatibus architecto temporibus provident quo? Repellat!</a>
+<a class="{{@root.prefix}}--link">Placeholder link</a>
```

Comment:
Do we have to move around the order...?

Reply:
I thought it made sense to group the valid/invalid states together, but I reverted the change